app/views/admin/sites/_form.html.erb

```diff
@@ -83,4 +83,5 @@
   <% unless @site.new_record? %>
     <%= link_to "Destroy Site", admin_site_path(@site), method: :delete, class: "btn btn-danger btn-lg" %>
   <% end %>
+  <br />
```

Comment:
Fixes a small layout issue where the update/delete buttons appear right at the bottom of the page, there's no leeway between that and the exact lower boundary of the page. It's really awkward lol.

Reply:
just to note we don't use a trailing slash on self closing tags since html5 :)

Reply:
oooo